### PR TITLE
feature/FLOW-1392 add CRUD functionality to dummy service

### PR DESCRIPTION
**Pull request by xiaoqianshi-dell**
Copied and modified code from dummy service to implement CRUD functionality to dummy service.

### CODE REVIEW

File: src/main/java/com/manywho/services/dummy/dummy/Account.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.manywho.services.dummy.dummy;
```
Comment (josecollazzi) [inline src/main/java/com/manywho/services/dummy/dummy/Account.java]: This class would be better organized in other domain directory like `com.manywho.services.dummy.account`

File: src/main/java/com/manywho/services/dummy/dummy/AccountDatabase.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
+package com.manywho.services.dummy.dummy;
```
Comment (josecollazzi) [inline src/main/java/com/manywho/services/dummy/dummy/AccountDatabase.java]: The same with this class (com.manywho.services.dummy.account)

File: src/main/java/com/manywho/services/dummy/repositories/AccountRepository.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
+package com.manywho.services.dummy.repositories;
```
Comment (josecollazzi) [inline src/main/java/com/manywho/services/dummy/repositories/AccountRepository.java]: This one also can go in that directory (com.manywho.services.dummy.account)

File: src/main/java/com/manywho/services/dummy/repositories/AccountRepository.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        String sql = "INSERT INTO account (id, name, company) VALUES (:id, :name, :company)";
+
+        try (Connection connection = sql2o.open()) {
+            int newAccountId = connection.createQuery(sql)
+                    .addParameter("name", account.getName())
+                    .addParameter("company", account.getCompany())
+                    .executeUpdate()
+                    .getResult();
+
+            account.setId("" + newAccountId);
```
Comment (josecollazzi) [inline src/main/java/com/manywho/services/dummy/repositories/AccountRepository.java]: The current line works, but String.valueOf() or Integer.toString() would be more explicit.

File: pom.xml [hunk 5]
```diff
@@ -54,6 +54,16 @@
             <artifactId>logstash-logback-encoder</artifactId>
             <version>6.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.sql2o</groupId>
+            <artifactId>sql2o</artifactId>
+            <version>1.5.4</version>
```
Comment (josecollazzi) [inline pom.xml]: can we use a newer version of this library?

File: pom.xml [hunk 6]
```diff
@@ -54,6 +54,16 @@
...
         </dependency>
+        <dependency>
+            <groupId>org.sql2o</groupId>
+            <artifactId>sql2o</artifactId>
+            <version>1.5.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.xerial</groupId>
+            <artifactId>sqlite-jdbc</artifactId>
+            <version>3.8.11.2</version>
```
Comment (josecollazzi) [inline pom.xml]: maybe is possible to use a new version for this one too.

**Review by josecollazzi — CHANGES_REQUESTED** [review]
Nice code, just minor changes required.

File: src/main/java/com/manywho/services/dummy/dummy/Account.java (hunk 1 above)
Comment (xiaoqianshi-dell) [reply src/main/java/com/manywho/services/dummy/dummy/Account.java]: Created a new account package and moved this file to there com.manywho.services.dummy.account

File: src/main/java/com/manywho/services/dummy/dummy/AccountDatabase.java (hunk 2 above)
Comment (xiaoqianshi-dell) [reply src/main/java/com/manywho/services/dummy/dummy/AccountDatabase.java]: Created a new account package and moved this file to there com.manywho.services.dummy.account

File: src/main/java/com/manywho/services/dummy/repositories/AccountRepository.java (hunk 4 above)
Comment (xiaoqianshi-dell) [reply src/main/java/com/manywho/services/dummy/repositories/AccountRepository.java]: I used String.valueOf()  initially then I saw this is used in demo service, so I copied it over. :D  

Changed it to String.valueOf() now.

File: pom.xml (hunk 5 above)
Comment (xiaoqianshi-dell) [reply pom.xml]: I checked the version on https://mvnrepository.com/  , changing it to 1.6.0

File: pom.xml (hunk 6 above)
Comment (xiaoqianshi-dell) [reply pom.xml]: I checked the version on https://mvnrepository.com/ , changing it to 3.32.3.2

**Review by josecollazzi — APPROVED** [review]
LGTM :100:

**Pull request merged**